### PR TITLE
Fixes so the hospitalization tests will run under devtools::test

### PR DESCRIPTION
**Pull request by jwills**
Without this, I get:

Joshs-MacBook-Air:hospitalization jwills$ Rscript -e 'devtools::test()'
Loading hospitalization
Testing hospitalization
✔ |  OK F W S | Context
✖ |   0 2     | test_hospitalization [0.5 s]
────────────────────────────────────────────────────────────────────────────────
test_hospitalization.R:12: failure: create_delay_frame works
`{ ... }` threw an error.
Message: could not find function "data.table"
Class:   simpleError/error/condition
Backtrace:
 1. testthat::expect_error(...) tests/testthat/test_hospitalization.R:12:2
 6. hospitalization::hosp_create_delay_frame(...) tests/testthat/test_hospitalization.R:13:4

test_hospitalization.R:16: error: create_delay_frame works
could not find function "data.table"
Backtrace:
 1. testthat::expect_lt(...) tests/testthat/test_hospitalization.R:16:2
 4. hospitalization::hosp_create_delay_frame(...) tests/testthat/test_hospitalization.R:17:4
────────────────────────────────────────────────────────────────────────────────

══ Results ═════════════════════════════════════════════════════════════════════
Duration: 0.6 s

OK:       0
Failed:   2
Warnings: 0
Skipped:  0

### CODE REVIEW

File: R/pkgs/hospitalization/tests/testthat/test_hospitalization.R [hunk 1]
```diff
@@ -1,9 +1,10 @@
 test_that("create_delay_frame works",{
+  require(data.table)
   X <- "input"
   varname <- "output"
   p_X <- 0.1
   X_pars <- c(10,1)
-  data_ <- data.frame(
```
Comment (jkamins7) [inline R/pkgs/hospitalization/tests/testthat/test_hospitalization.R:6]: This is definitely not an appropriate change... I would rather count this test as failed than require the input to be data tables

**Comment by jwills** [comment]
Sent an update to use `data.frame`...but the test still won't pass unless `data.table` is loaded, since the function, as written, requires it to exist in order to work. Is it better to move the `require` inside of `hosp_create_delay_frame`?

**Comment by jkamins7** [comment]
long term, it's inappropriate to use require in package code.  short term, yes moving the require inside is a reasonable way to resolve this test

**Comment by jwills** [comment]
Current version passes the test and just uses the fully-qualified function name to do it, as a good package function should.

**Pull request merged**